cmd/tsbs_generate_data/main.go

```diff
@@ -169,8 +173,13 @@ func init() {
 
 	flag.IntVar(&debug, "debug", 0, "Debug printing (choices: 0, 1, 2). (default 0)")
 
-	flag.UintVar(&interleavedGenerationGroupID, "interleaved-generation-group-id", 0, "Group (0-indexed) to perform round-robin serialization within. Use this to scale up data generation to multiple processes.")
-	flag.UintVar(&interleavedGenerationGroups, "interleaved-generation-groups", 1, "The number of round-robin serialization groups. Use this to scale up data generation to multiple processes.")
+	flag.UintVar(&interleavedGenerationGroupID,
+		"interleaved-generation-group-id", 0,
```

Comment:
move these two args to the previous line, leaving just the description on a separate line.

same for the flag below too

Reply:
Yes, agree